gulpfile.js

```diff
@@ -769,14 +770,14 @@ gulp.task('axe:components', callback => {
   return axe(Object.assign({}, axeOptions, {
     saveOutputIn: 'components.json',
     urls: [
-      'build/components/preview/!(input*|*layout*).html'
+      'build/components/preview/!(input*|*layout*|preview*|textarea*|teaser--*).html'
```

Comment:
Aren't teasers components?

Reply:
That is very much correct! It was included in the original gulp task as well and so I tossed it out at first, but... 

A teaser is an `li` element which is collated in the overview by this function:

https://github.com/StadGent/fractal_styleguide_gent-base/blob/39c6e0a7701c21080bf7473e3b0c5f818d93afec/components/31-molecules/teaser/teaser.config.js#L4-L15

However, you can access the teasers individually which leads to a axe violation stating that an `li` should always be contained in a `ul`. Hence why the `teaser--` urls are skipped.